src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,14 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  Array.prototype.indexOf2 = function(searchElement, fromIndex = 0) {
+    for (let i = 0; i < this.length; i++) {
```

Comment:
так краще і логічніше
```suggestion
    for (let i = fromIndex; i < this.length; i++) {
```

Reply:
Нужно по разному попробовать.)

Reply:
так, але ти робиш не потрібні ітерації циклу.

якщо в тебе масив буде з 10 000 елементів і fromIndex буде дорівнювати 5 000, ти просто зробиш зайвих 5 000 ітерацій

Reply:
Я понял. Спасибо)

Reply:
Думал, разницы особой нет. А вот оно, как.

Reply:
Виправляй)

Reply:
Я уже исправил!

Reply:
Ой. Запушить забыл. Сори)